projects/plugins/jetpack/modules/simple-payments/simple-payments.php

```diff
@@ -93,25 +93,25 @@ private function register_scripts_and_styles() {
 		 *
 		 * @see https://developer.paypal.com/docs/integration/direct/express-checkout/integration-jsv4/add-paypal-button/
 		 */
-		wp_register_script(
+		wp_register_script( // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
 			'paypal-checkout-js',
 			'https://www.paypalobjects.com/api/checkout.js',
 			array(),
-			JETPACK__VERSION,
+			null,
 			true
 		);
-		wp_register_script(
+		wp_register_script( // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
 			'jetpack-paypal-express-checkout',
 			plugins_url( '/paypal-express-checkout.js', __FILE__ ),
 			array( 'jquery', 'paypal-checkout-js' ),
-			JETPACK__VERSION,
+			null,
```

Comment:
This one and the following one I think should use `JETPACK__VERSION`. It was only the one that's fetching from https://www.paypalobjects.com/api/checkout.js instead of a file within Jetpack that I was referring to before.